src/react/boosts/BoostTextComponents.tsx

```diff
@@ -3,28 +3,45 @@ import { ModalHeading, ModalContent } from "@joincivil/components";
 
 export const WhyEthModalText: React.FunctionComponent = props => (
   <>
-    <ModalHeading>
-      Why ETH?
-    </ModalHeading>
-    <ModalContent>
-      <p>100% of your ETH will go right into the newsroom’s wallet. This way, the Newsroom get’s your full support.</p>
-      <p>You’ll to use Ethereum cryptocurrency (ETH) in a digital wallet like MetaMask in order to continue. Currently it’s not possible to use other cryptocurrencies, or in dollars or other fiat currencies.</p>
-      <p>There is a very small network transaction cost for sending Ethereum out of your wallet, usually it’s a matter of a few cents. This cost goes to the Ethereum miners who perform the computational work for your content to be included on the Ethereum blockchain.</p>
-      <a href="">Learn more about how to get ETH and support Boosts on our FAQ.</a>
-    </ModalContent>
+    <ModalHeading>Why ETH?</ModalHeading>
+    <ModalContent>100% of your ETH will go right into the newsroom’s wallet. This way, the Newsroom get’s your full support.</ModalContent>
```

Comment:
Couple typos, not sure if these are here from before or if you got fresh from invision and typos haven't been fixed there (I commented on them I think Rob got them not sure) but there's a couple typos

- `get's` > `gets` in this paragraph
- `you'll to use` > `you'll have to use` next paragraph
- `cryptocurrencies, or in dollars or...` >`cryptocurrencies, dollars, or...` next paragraph

If you haven't I'd copy paste fresh from invision, there may be some other copy cleanup.